backend/src/test/java/com/daedan/festabook/place/controller/PlaceControllerTest.java

```diff
@@ -468,4 +481,79 @@ class getPlaceWithDetailByPlaceId {
                     .body("message", equalTo("존재하지 않는 플레이스입니다."));
         }
     }
+
+    @Nested
+    class deleteByPlaceId {
+
+        @Test
+        void 성공() {
+            // given
+            Organization organization = OrganizationFixture.create();
+            organizationJpaRepository.save(organization);
+
+            Place place = PlaceFixture.create(organization);
+            placeJpaRepository.save(place);
+
+            PlaceDetail placeDetail = PlaceDetailFixture.create(place);
+            placeDetailJpaRepository.save(placeDetail);
+
+            Device device1 = DeviceFixture.create();
+            Device device2 = DeviceFixture.create();
+            deviceJpaRepository.saveAll(List.of(device1, device2));
+
+            PlaceImage placeImage1 = PlaceImageFixture.create(place);
+            PlaceImage placeImage2 = PlaceImageFixture.create(place);
+            placeImageJpaRepository.saveAll(List.of(placeImage1, placeImage2));
+
+            PlaceAnnouncement placeAnnouncement1 = PlaceAnnouncementFixture.create(place);
+            PlaceAnnouncement placeAnnouncement2 = PlaceAnnouncementFixture.create(place);
+            placeAnnouncementJpaRepository.saveAll(List.of(placeAnnouncement1, placeAnnouncement2));
+
+            PlaceFavorite placeFavorite1 = PlaceFavoriteFixture.create(place, device1);
+            PlaceFavorite placeFavorite2 = PlaceFavoriteFixture.create(place, device2);
+            placeFavoriteJpaRepository.saveAll(List.of(placeFavorite1, placeFavorite2));
+
+            // when & then
+            RestAssured
+                    .given()
+                    .header(ORGANIZATION_HEADER_NAME, organization.getId())
+                    .when()
+                    .delete("/places/{placeId}", place.getId())
+                    .then()
+                    .statusCode(HttpStatus.NO_CONTENT.value());
+
+            assertSoftly(s -> {
+                s.assertThat(placeJpaRepository.findById(place.getId())).isEmpty();
+
+                s.assertThat(placeDetailJpaRepository.findById(placeDetail.getId())).isEmpty();
+
+                s.assertThat(placeImageJpaRepository.findById(placeImage1.getId())).isEmpty();
+                s.assertThat(placeImageJpaRepository.findById(placeImage2.getId())).isEmpty();
+
+                s.assertThat(placeAnnouncementJpaRepository.findById(placeAnnouncement1.getId())).isEmpty();
+                s.assertThat(placeAnnouncementJpaRepository.findById(placeAnnouncement2.getId())).isEmpty();
+
+                s.assertThat(placeFavoriteJpaRepository.findById(placeFavorite1.getId())).isEmpty();
+                s.assertThat(placeFavoriteJpaRepository.findById(placeFavorite2.getId())).isEmpty();
+            });
+        }
+
+        @Test
+        void 성공_place가_존재하지_않는_경우_204를_응답한다() {
```

Comment:
동사가 아닌 명사로 하기로 했으니, 

```suggestion
        void 성공_존재하지_않는_Place인_경우도_204_응답() {
```

어떤가요?!